docs/api/model_selection/cross_validation.rst

```diff
@@ -41,7 +41,7 @@ In the following example, we show how to visualize cross-validated scores for a
     X, y = load_occupancy()
 
     # Create a cross-validation strategy
-    cv = StratifiedKFold(n_splits=12, random_state=42)
+    cv = StratifiedKFold(n_splits=12, shuffle=True, random_state=42)
```

Comment:
The docs were raising an exception because a random state was supplied without the shuffle argument set to true. The result was that the images were not shown in the build docs. In fact, they're not shown in our [current docs](https://www.scikit-yb.org/en/latest/api/model_selection/cross_validation.html#classification). Adding `shuffle=True` prevented the exception from being raised.